addons/web/static/src/search/control_panel/control_panel.js

```diff
@@ -606,8 +617,8 @@ export class ControlPanel extends Component {
     _sortEmbeddedActionDrop({ element, previous }) {
         const order = this.state.embeddedInfos.embeddedActions
             .map((el) => el.id)
-            .filter((el) => el !== false);
-        const elementId = Number(element.dataset.id);
+            .filter((el) => el !== false || this.isMainEmbeddedActionOrderModifiable);
```

Comment:
then no need to filter if `this.isMainEmbeddedActionOrderModifiable` is truly, right? 🧐

Reply:
Yep, you're right :)